config/monitoring/role_binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: openshift-monitoring
+- kind: ServiceAccount
+  name: prometheus-user-workload
+  namespace: openshift-user-workload-monitoring
```

Comment:
What is the implication of always setting up this RoleBinding to **both** the usual `openshift-monitoring` ServiceAccount and the user-workload ServiceAccount?

Reply:
Always adding the role binding would mean that if the user created a service monitor, like SD does on it's own, the openshift in built monitoring will start capturing metrics which the user might not really want.
so making sure we add the service monitors and rbac when exportMetrics is true keeps things more predictable.